templates/pages/blog.html

```diff
@@ -13,7 +13,7 @@
     <h1 class="page-heading">{{ blog.name }}</h1>
 
     {{#each blog.posts}}
-        {{> components/blog/post post=this settings=../settings}}
```

Comment:
No `settings=../settings` like everywhere else?

Reply:
Turns out `components/blog/post` doesn't use `settings` in the first place.